website/static/js/myProjects.js

```diff
@@ -1777,31 +1777,7 @@ var Filters = {
  */
 var Information = {
     view : function (ctrl, args) {
-        function categoryMap(category) {
-            // TODO, you don't need to do this, CSS will do this case change
-            switch (category) {
-                case 'analysis':
-                    return 'Analysis';
-                case 'communication':
-                    return 'Communication';
-                case 'data':
-                    return 'Data';
-                case 'hypothesis':
-                    return 'Hypothesis';
-                case 'methods and measures':
-                    return 'Methods and Measures';
```

Comment:
We made an explicit decision to not automatically capitalize the category identifiers and instead use a mapping (https://github.com/CenterForOpenScience/osf.io/blob/459172cde60ebe0d5f9d93d6ac56c5cc2e47ebc1/website/settings/defaults.py#L209-L221). 

The identifiers are meant to be stable, whereas the human-readable representation is meant to be change-able. We may, for example, change "Methods and Measures" to "Methods".

Pedantry aside, using `.text-capitalize` is probably fine for the time being.